media-autobuild_suite.bat

```diff
@@ -1181,6 +1181,32 @@ if %timeStampF%==2 set "timeStamp=n"
 if %timeStampF% GTR 2 GOTO timeStamp
 if %deleteINI%==1 echo.timeStamp=^%timeStampF%>>%ini%
 
+:ccache
+if %ccacheINI%==0 (
+    echo -------------------------------------------------------------------------------
+    echo -------------------------------------------------------------------------------
+    echo.
+    echo. Use ccache when compiling?
+    echo. Experimental.
+    echo. Speeds up rebuilds and recompilations, but requires the files to be
+    echo. compiled at least once before any effect is seen
+    echo. 1 = Yes
+    echo. 2 = No
+    echo.
+    echo -------------------------------------------------------------------------------
+    echo -------------------------------------------------------------------------------
+    set /P buildwithccache="Use ccache: "
+) else set buildwithccache=%ccacheINI%
+
+if "%buildwithccache%"=="" GOTO ccache
+if %buildwithccache%==1 (
+    set "ccache=y"
+    set mingwpackages=%mingwpackages% ccache
```

Comment:
ccache is being added here

Reply:
You can always install it. It's just a 200KB package.